wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -936,6 +936,13 @@ impl PhysicalDeviceProperties {
             extensions.push(vk::KhrDrawIndirectCountFn::name());
         }
 
+        // Require `VK_EXT_subgroup_size_control` if the associated feature was requested
+        // Even though Vulkan 1.3 has promoted the extension to core, we must require the extension to avoid
```

Comment:
Looks like this won't do and we have to use `PhysicalDeviceVulkan13Features` after all. @teoxoy Thoughts?